<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DummyProviderParametersControlWidget</class>
 <widget class="QWidget" name="DummyProviderParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DummyProviderParameters</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Path</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="pathEdit">
       <property name="toolTip">
         <string>Path</string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Unknown Value</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="unknownValueSpinBox">
       <property name="toolTip">
         <string>Unknown Value</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>3000000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Plane Zero</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="planeZeroSpinBox">
       <property name="toolTip">
         <string>Plane Zero</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>3000000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>Plane Max</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="planeMaxSpinBox">
       <property name="toolTip">
         <string>Plane Max</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>3000000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>65536</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>pathEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DummyProviderParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>unknownValueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DummyProviderParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>planeZeroSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DummyProviderParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>planeMaxSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DummyProviderParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
